src/QirRuntime/samples/StandaloneInputReference/qsharp/qir-standalone-input-reference.qs

```diff
@@ -4,6 +4,28 @@
     open Microsoft.Quantum.Canon;
     open Microsoft.Quantum.Intrinsic;
 
+    function ArrayToString<'T> (array : 'T[]) : String
+    {
+        mutable first = true;
+        mutable itemsString = "[";
+        for item in array
+        {
+            if (first)
+            {
+                set itemsString = itemsString + $"{item}";
+            }
+            else
+            {
+                set itemsString = itemsString + $", {item}";
+            }
+
+            set first = false;
```

Comment:
Consider moving `set first = false;` to inside of `if (first) { .. }`.

Reply:
Moved to inside the if-statement.